code/modules/mining/equipment/explorer_gear.dm

```diff
@@ -83,11 +87,13 @@
 	armor = list(MELEE = 70, BULLET = 40, LASER = 10, ENERGY = 10, BOMB = 50, BIO = 100, RAD = 100, FIRE = 100, ACID = 100)
 	resistance_flags = FIRE_PROOF | LAVA_PROOF
 
-/obj/item/clothing/head/helmet/space/hostile_environment/New()
-	..()
-	AddComponent(/datum/component/spraycan_paintable)
+/obj/item/clothing/head/helmet/space/hostile_environment/Initialize(mapload)
+	. = ..()
 	update_icon()
 
+/obj/item/clothing/head/helmet/space/hostile_environment/ComponentInitialize()
+	AddComponent(/datum/component/spraycan_paintable)
```

Comment:
Be sure to apply the above reasoning here as well then.

Reply:
Whoops